src/syzygy/tbprobe.h

```diff
@@ -30,19 +30,19 @@ class Position;
 namespace Stockfish::Tablebases {
 
 enum WDLScore {
```

Comment:
I think the below are better and more concise as they are.  Comments are not necessarily meant to be full sentences.